File: RA 1.1/Ejercicio_1.py
class Persona: 
    def __init__(self, nombre, edad):
        self.nombre= nombre 
        self.edad= edad

    def saludar(self): 
        print(f"hola, mi nombre es {self.nombre} y tengo {self.edad} años")

    def cumple(self): 
        self.edad= self.edad+1 
        print(f"{self.nombre} ha cumplido años! Ahora tiene {self.edad} años")

persona1= Persona("Orlando", 17)
persona2= Persona("Marco", 16)

persona1.saludar()
persona2.saludar()

persona1.cumple()

File: RA 1.1/Ejercicio_1.cpp
#include <iostream>
using namespace std; 

class Persona {
    protected: 
        string nombre; 
        int edad; 

    public: 
        Persona(string _nombre, int _edad): nombre(_nombre), edad(_edad) {}
        ~Persona() {}

        void saludar() {
            cout<<"hola, mi nombre es "<<this->nombre<<" y tengo "<<this->edad<<"anios"<<endl;
        }

        void cumple() { 
            this->edad += 1;
            cout<<this->nombre<<" ha cumplido anios! Ahora tiene "<<this->edad<<" anios"<<endl;
        }
};

int main() {
    Persona persona1("Orlando", 17);
    Persona persona2("Marco", 16);

    persona1.saludar(); 
    persona2.saludar(); 

    persona1.cumple();

    return 0;
}